libs/community/langchain_community/vectorstores/databricks_vector_search.py

```diff
@@ -321,6 +324,134 @@ def similarity_search_with_score(
         )
         return self._parse_search_response(search_resp)
 
+    def _select_relevance_score_fn(self) -> Callable[[float], float]:
+        """
+        The 'correct' relevance function
+        may differ depending on a few things, including:
+        - the distance / similarity metric used by the VectorStore
+        - the scale of your embeddings (OpenAI's are unit normed. Many others are not!)
+        - embedding dimensionality
+        - etc.
+        """
+
+        def _db_relevance_score_fn(distance: float) -> float:
+            """
+            Scores returned by the Databricks Vector Search API are
+            normalized cosine similarity scores on a scale [0, 1] -
+            1 means most similar
+            """
+
+            return distance
```

Comment:
nit: Instead of having this nested function, can we convert this to a class-level static method similar to [here](https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/vectorstores/elasticsearch.py#L699)?

Reply:
Makes sense. I've changed it.